Sophos/sophos firewall/ingest/parser.yml

```
@@ -12,6 +12,9 @@ pipeline:
 stages:
   core:
     actions:
+      - set:
+          log.hostname: "{{kv_event.message.device_serial_id}}"
```

[`observer.serial_number`](https://www.elastic.co/guide/en/ecs/8.17/ecs-observer.html#field-observer-serial-number) is more suitable:
```suggestion
          observer.serial_number: "{{kv_event.message.device_serial_id}}"
```